exporter/http.go

```diff
@@ -13,6 +14,13 @@ import (
 	log "github.com/sirupsen/logrus"
 )
 
+type discoveryTargetList []*discoveryTargetGroup
+
+type discoveryTargetGroup struct {
```

Comment:
the two types aren't really used anywhere else, can you just inline this in line 133?